package.json

```
@@ -0,0 +1,24 @@
+{
+  "name": "mdlinks",
+  "version": "1.0.0",
+  "description": "Proyecto mdlinks de Laboratoria",
+  "main": "app.js",
+  "bin": {
+    "mdlinks":"cli.js"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

recuerda que el apartado de los scripts se utiliza para ejecutar los comandos útiles en el desarrollo de la librería
```suggestion
    "test": "jest"
```